timeline.js

```diff
@@ -3,22 +3,39 @@ import { db } from "./dbInstance.js";
 class ReaderTimeline extends HTMLElement {
   constructor() {
     super();
-    this.actorUrls = [
-      "https://staticpub.mauve.moe/about.jsonld",
-      "https://hypha.coop/about.jsonld",
-      "https://prueba-cola-de-moderacion-2.sutty.nl/about.jsonld",
-    ];
-    this.processedNotes = new Set(); // To keep track of notes already processed
+    this.processedNotes = new Set(); // To keep track of already processed notes
   }
 
   connectedCallback() {
-    this.initTimeline();
+    this.initializeDefaultFollowedActors().then(() => this.initTimeline());
+  }
+
+  async initializeDefaultFollowedActors() {
+    const defaultActors = [
+      "https://social.dp.chanterelle.xyz/v1/@announcements@social.dp.chanterelle.xyz/",
+      "https://hypha.coop/about.jsonld",
+      "https://sutty.nl/about.jsonld",
+      // "https://akhilesh.sutty.nl/about.jsonld",
+      // "https://staticpub.mauve.moe/about.jsonld",
+    ];
+
+    // Check if followed actors have already been initialized
+    const hasFollowedActors = await db.hasFollowedActors();
+    if (!hasFollowedActors) {
+      for (const actorUrl of defaultActors) {
```

Comment:
```suggestion
      await Promise.all(defaultActors.map(async (actorUrl) => {
```

This will make the ingestion happen in paralell which would be faster